tox.ini

```diff
@@ -4,8 +4,7 @@
 # and then run "tox" from this directory.
 
 [tox]
-#envlist = py27, py32, py34, py35
-envlist = py27, py35
+envlist = py27, py3
```

Comment:
We have been having this debate ourselves. While the py3 target is easier it is also useful to assert your support py34, py35, py36, py37 or whichever ones you believe should work. 

You might consider adding these additional targets.